server/views/pages/goal/create/newIndex.njk

```diff
@@ -0,0 +1,190 @@
+
+              {% for futureGoalTargetDate in futureGoalTargetDates %}
+                {% set targetCompletionDateItems = (targetCompletionDateItems.push({
+                  "value": futureGoalTargetDate.value,
+                  "text": futureGoalTargetDate.text
+                  }), targetCompletionDateItems)
+                %}
+              {% endfor %}
+
+              {% set anotherDateHtml %}
+                {{ govukDateInput({
+                  id: "another-date",
+                  fieldset: {
+                    legend: {
+                      text: "What date are they aiming to achieve this by?",
+                      classes: "govuk-visually-hidden"
+                    }
+                  },
+                  hint: {
+                    text: "For example, 27 3 2007"
+                  },
+                  items: [
+                    {
+                      id: 'another-date-day',
+                      label: 'Day',
+                      name: 'targetCompletionDate-day',
+                      classes: 'govuk-input--width-2',
+                      value: form['targetCompletionDate-day']
+                    },
+                    {
+                      id: 'another-date-month',
+                      label: 'Month',
+                      name: 'targetCompletionDate-month',
+                      classes: 'govuk-input--width-2',
+                      value: form['targetCompletionDate-month']
+                    },
+                    {
+                      id: 'another-date-year',
+                      label: 'Year',
+                      name: 'targetCompletionDate-year',
+                      classes: 'govuk-input--width-4',
+                      value: form['targetCompletionDate-year']
+                    }
+                  ]
+                }) }}
+              {% endset -%}
+
+              {% set targetCompletionDateItems = (targetCompletionDateItems.push({
+                "value": "another-date",
+                "text": "set another date",
+                "conditional": {
+                  "html": anotherDateHtml
+                }
+                }), targetCompletionDateItems)
+              %}
+
+              {{ govukRadios({
+                name: "targetCompletionDate",
+                id: "targetCompletionDate",
+                value: form.targetCompletionDate,
+                fieldset: {
+                  legend: {
+                    text: "When are they aiming to achieve this by?",
+                    classes: "govuk-fieldset__legend--s"
+                  }
+                },
+                items: targetCompletionDateItems,
+                errorMessage: errors | findError('targetCompletionDate')
+              }) }}
+
+            <div class="govuk-form-group">
+              {% set titleLabel = 'Step ' + form.stepNumber %}
+              {{ govukCharacterCount({
+                name: "title",
+                id: "title",
+                value: form.title,
```

Comment:
Presumably this wants to be the title of the step, rather than the title of the goal?
At the moment both fields are using the same property in the form object

Given we've not designed the form object yet, could you change it to `form.stepTitle`, but add a todo to it to reflect a specific step's title? (ie. we might use an array in the form object, so this form field will need to index into the array)